0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,82 +1,91 @@
 ---
 title: Interledger Architecture
-draft: 1
+draft: 2
 ---
 # Interledger Architecture
 
 Interledger provides for secure payments across multiple assets on different ledgers. The architecture consists of a conceptual model for interledger payments, a mechanism for securing payments, and a suite of protocols that implement this design.
 
 The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite.
 
-Colloquially the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
+Colloquially, the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
 
 The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
 
 ## Interledger Model
 
-![Interledger Model: Sender, Connectors, Ledgers, Receiver](../shared/graphs/interledger-model.svg)
+![Interledger Model: Sender, Connectors, Receiver](../shared/graphs/interledger-model.svg)
 
-### Ledgers
+### Connectors
 
-*Ledgers* are stateful systems that track the ownership of *assets*. Ledgers contain buckets of assets known as *accounts* and record *transfers* between them. Each account has a *balance*, which is the amount of the ledger's assets the account holds. Account balances may be positive or negative, representing assets or liabilities.
+A *connector* is a system that provides the service of forwarding Interledger packets towards the destination provided in the packet. Packets are forwarded across connections between senders, connectors and receivers, called *accounts*.
```

Comment:
What are "called accounts"? It's hard to understand what in this sentence that is referring to.

Reply:
changing this to 'across connections (*accounts*) between ...'